dev/dll/XamlControlsResources.cpp

```diff
@@ -87,6 +92,61 @@ void XamlControlsResources::UpdateSource()
     ThemeDictionaries().Clear();
 
     Source(uri);
+
+    // Hacky workaround for a XAML compiler bug:
+    // Assigning nullable primitive types from XAML fails with disabled XAML metadata reflection on older versions.
+    // The MUXC AcrylicBrush's TintLuminosityOpacity is a nullable double though and needs to be set.
+    // Solution: Load theme resources and edit brushes manually.
+    // Since something must go horribly wrong for those lookups to fail, we just assume they exist.
+    if (SharedHelpers::Is19H1OrHigher())
```

Comment:
Why is there a 19H1+ condition?

Reply:
TintLuminosityOpacity is only present on 19H1+ and we also have a separate themeresources file because of that. To not interfere with the RS5- brushes and save performance, we are only updating the brushes when on 19H1+. Of course, we could run it in every case as the MUXC brushes guard against it but we don't need to.

Reply:
can you add a comment in the 19h1 theme resource file that tint opacity is being set in code ?

Reply:
Good idea, added a comment for that. Can you rerun CI @ranjeshj ?